templates/experiments/components/experiment_version_details_content.html

```diff
@@ -1,16 +1,53 @@
 <div class="modal-header">
     <div class="modal-title mb-5" id="detailsModalLabel">
-        <h3 class="text-base text-xl font-bold">Details for {{ experiment }}</h3>
+        <h3 class="text-base text-xl font-bold mb-2">Details for {{ experiment }}</h3>
+        {% url 'experiments:set-default-experiment' team_slug=team.slug experiment_id=experiment.working_version_id version_number=experiment.version_number as update_default_url%}
+        <form method="post" action="{{ update_default_url }}">
+            {% csrf_token %}
+            <button type="submit" class="btn btn-sm btn-primary" {% if experiment.is_default_version %}disabled{% endif %}>
+                Make default version
+            </button>
+        </form>
     </div>
-    {% url 'experiments:set-default-experiment' team_slug=team.slug experiment_id=experiment.working_version_id version_number=experiment.version_number as form_action_url%}
-    <form method="post" action="{{ form_action_url }}">
-        {% csrf_token %}
-        <button type="submit" class="btn btn-sm btn-primary" {% if experiment.is_default_version %}disabled{% endif %}>
-            Make default version
-        </button>
+    <form class="my-2" x-data="{
+                               editActive: false,
+                               descriptionText: '{{ experiment.version_description|escapejs }}',
+                               previousdescriptionText: '',
+                               cancelEdit(){
+                               this.editActive = false;
+                               this.descriptionText = this.previousdescriptionText;
+                               },
+                               concludeEdit(){
+                               this.editActive = false;
+                               this.previousdescriptionText = this.descriptionText;
+                               }
+                               }"
+          x-init="previousdescriptionText = descriptionText">
+        <span
+            x-show="!editActive"
+            @click="editActive = true"
+            class="hover:bg-base-200 hover:cursor-pointer text-sm p-2 rounded"
+            x-text="descriptionText">
+        </span>
+        <textarea name="description" x-show="editActive" x-model="descriptionText" class="textarea textarea-bordered text-sm w-full"></textarea>
+        <div class="btn-group">
+            <button
+                type="button"
+                x-show="editActive"
+                class="hover:bg-base-200 p-1 rounded"
```

Comment:
This should use one of the `btn` classes rather than custom styling if possible.